py4cast/lightning.py

```diff
@@ -893,14 +966,16 @@ def on_test_start(self):
                 ),
             ]
 
-    def test_step(self, batch: ItemBatch, batch_idx):
+    def test_step(self, batch: ItemBatch, batch_idx: int):
         """
         Run test on single batch
         """
```

Comment:
Docstring formating.
```suggestion
        """Runs test on single batch
        """
```